test/Microsoft.Health.Fhir.Tests.E2E/Rest/SuppressExecutionContextHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest
+{
+    internal class SuppressExecutionContextHandler : DelegatingHandler
```

Comment:
Add a comment pointing to https://github.com/aspnet/AspNetCore/issues/7975#issuecomment-481536061